src/components/Card/Card.tsx

```diff
@@ -118,17 +117,31 @@ export default class Card extends React.Component<CardProps, {
 
       const { stargazers_count, pushed_at } = repoData;
 
-      if (this.state.stars !== stargazers_count) {
+      if (this.state.stars !== stargazers_count && this.props.CONFIG.visual.stars) {
         this.setState({ stars: stargazers_count }, () => {
           console.log(`Stars updated to: ${this.state.stars}; updating localstorage.`);
-          this.installExtension();
+          switch (this.props.type) {
+          case "extension":
+            this.installExtension();
+            break;
+          case "theme":
+            this.installTheme();
+            break;
+          }
         });
       }
 
       if (this.state.lastUpdated !== pushed_at) {
         this.setState({ lastUpdated: pushed_at }, () => {
           console.log(`New update pushed at: ${this.state.lastUpdated}; updating localstorage.`);
-          this.installExtension();
+          switch (this.props.type) {
```

Comment:
Could it ever run both of these? May want to tweak the if statements/logic flow so it can't run on both cases.

Reply:
I'm pretty sure not, because once the star is updated it would update localStorage also, which makes the new update case false. But we can return the function immediately after updating stars.

Reply:
Fixed.